### PR TITLE
To Do List Structure

### PR DESCRIPTION
**Pull request by lily-coder**
In this section I have created the to do list
The user can add items to the list, check them when done and delete them

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,48 @@
...
+- Javascript
+- Webpack
+
+### Setup
+To get a local copy up and running follow these simple example steps.
+
+- Clone the repository to your computer using this [list](https://github.com/lily-coder/to-do-list.git)
+- Cd into the folder
+- Open index.html with live server
+- Or you can Fork it instead, and create a pull request to suggest some changes
```
Comment (bellom) [inline README.md:22]: We are using `Webpack` to run this project, our webpack dev server should be working properly, users should be able to run `npm start` command successfully, in your case for some reason, it is not.

Let's try to fix the webpack, then you can now add these steps👇 

```
- git clone `your repo`
- cd into the folder
- git checkout `branch-name`
- run `npm i`
- run `npm start`
```

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @lily-coder  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well with the design, setting up linter and good commit messages 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 

_Kindly check the comments under the review._

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lily-coder** [comment]
I have implemented the changes as requested by the reviewer. The web dev server is working well on running npm start

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @lily-coder , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [ ] As the previous TSE requested, kindly update the README file with the steps he gave you. I can see the commit you made but it does not reflect on your README file. Kindly double-check on this.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Comment by lily-coder** [comment]
Hello @mimipeshy , The instructions are not very clear. would you mind to elaborate please? Because the README file is updated with the changes the previous reviewer suggested and it shows properly. 
image
as in the image above

**Comment by bellom** [comment]
> Hello @mimipeshy , The instructions are not very clear. would you mind to elaborate please? Because the README file is updated with the changes the previous reviewer suggested and it shows properly.
> image
> as in the image above

@lily-coder I think you missed `npm i`

**Comment by lily-coder** [comment]
> > Hello @mimipeshy , The instructions are not very clear. would you mind to elaborate please? Because the README file is updated with the changes the previous reviewer suggested and it shows properly.
> > image
> > as in the image above
> 
> @lily-coder I think you missed `npm i`

I get it now. Thank you so much @bellom

**Review by dasileker — APPROVED** [review]
### **Hi @lily-coder** 

### **Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**